internal/engine/experiment/stunreachability/stunreachability.go

```diff
@@ -83,12 +102,8 @@ func (m *Measurer) Run(
 func (tk *TestKeys) run(
 	ctx context.Context, config Config, sess model.ExperimentSession,
 	measurement *model.Measurement, callbacks model.ExperimentCallbacks,
+	endpoint string,
 ) error {
-	const defaultAddress = "stun.l.google.com:19302"
-	endpoint := string(measurement.Input)
-	if endpoint == "" {
-		endpoint = defaultAddress
-	}
```

Comment:
As mentioned, now the static input comes from the input loader rather than being hardcoded here. There are two advantages of this approach:

1. now `ooniprobe` can see and register the URLs in advance
2. we can have many static default input URLs instead than just one